### ISSUE TITLE
migrated README files still point to the old repository

### ISSUE THREAD

**Issue by dandhlee** [GoogleCloudPlatform/python-docs-samples#10628]
Looks like we need to perform a readme review on some of the migrated samples, Yu-Han helped mention that there might be a few more that still point to the `googleapis/python-*` repos. https://github.com/GoogleCloudPlatform/python-docs-samples/pull/10200#issuecomment-1721785280

**Comment by dandhlee** [comment]
I'll clear the assignee for now in case anyone wants to help update this :)

**Comment by carlosm00** [comment]
Hi team,
I'd love to collaborate, but I am not sure if I am right, so please, correct me if I am wrong. 
By reading https://github.com/GoogleCloudPlatform/python-docs-samples/pull/10200#pullrequestreview-1629615005, I understand that `https://github.com/googleapis/python-*` entries should be changed to `https://github.com/GoogleCloudPlatform/python-docs-samples`. I might use the list of merged migrations from [pull requests that are closed, merged and are named after 'migrate code from googleapis'](https://github.com/GoogleCloudPlatform/python-docs-samples/pulls?q=is%3Apr+is%3Aclosed+migrate+code+from+googleapis+is%3Amerged). 
If this is all correct, I will more than gladly do this task :D
Regards.

**Comment by carlosm00** [comment]
Hi @dandhlee,
Would you mind checking my last comment?
Best regards.

**Comment by iennae** [comment]
@carlosm00 yes, you've caught the gist. things that are in READMEs that point to the old location within the gooleapis org should point to GoogleCloudPlatform. In some cases, samples may have been refactored or updated but the READMEs are still stale. Thanks for any help you want to provide.
